genesis/sensors/tactile.py

```diff
@@ -17,6 +18,8 @@ class RigidContactSensor(Sensor):
     """
     Sensor that returns bool based on whether associated RigidLink is in collision.
 
+    read() -> np.ndarray of shape (batch_size,) of bool values
```

Comment:
What is this comment? Formatting is weird and this should not be part of high-level documentation. Maybe something like `One can retrieve the current value of a sensor by calling the method 'read'. See corresponding documentation for details.`